### PR TITLE
Add shared sessions boolean to the Client Config table

### PR DESCRIPTION
**Pull request by achievedaniel**
## Summary

Adding a shared sessions boolean for the auth oath project

## Related issue(s)

department-of-veterans-affairs/va.gov-team/75380

## Testing done

- [ x ] *New code is covered by unit tests*

## What areas of the site does it impact?
Client Config table adding another column.

## Acceptance criteria

- [ x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x ]  No error nor warning in the console.
- [x ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: app/models/sign_in/client_config.rb
Comment (rileyanderson) [reply app/models/sign_in/client_config.rb:19]: I think this is fine. `presence` won't work because it will reject `false` and this `inclusion` will catch `nil` but allow `false`. I agree that `nil` _should_ probably never happen especially with a default value but who knows 😱

Comment (bosawt) [reply app/models/sign_in/client_config.rb:19]: yeah this is fine (but of course I'd say that because this is a pattern I follow, ha). The reason to do this is just that I prefer having errors caught at the app level before getting to the database level, even if the database would definitely catch this as well

**Review by bosawt — CHANGES_REQUESTED** [review]
Looks good, I'm only setting this to 'request_changes' for now is to intentionally block this until the schema changes are merged we've proved they are on all the stacks

File: spec/models/sign_in/client_config_spec.rb
Comment (achievedaniel) [reply spec/models/sign_in/client_config_spec.rb:182]: Yes I was thinking along the same lines but looks like that the spec doesn't cover other scenarios.

**Review by rileyanderson — APPROVED** [review]

**Review by bosawt — APPROVED** [review]
Looks like migrations have been run, so this should be good to go

**Pull request merged**
